README.md

```diff
@@ -0,0 +1,74 @@
+<div align="center">
+
+# apc.nvim
+
+[About](#about) • [Install](#install) • [Usage](#usage) • [Contribute](#contribute)
+
+</div>
+
+---
+
+This Neovim plugin is written in Lua and provides daily information from [apeoplescalendar.org](https://www.apeoplescalendar.org/).
+
+## About
+
+A People's Calendar (aPC) is a project that seeks to promote the worldwide history of working class movements and liberation struggles in the form of a searchable "On This Day" calendar.
+
+This repository hosts the aPC neovim plugin, implemented in Lua.
+
+This project is open source and the information contained in our event database is non-proprietary and will _always_ be freely available for users, developers, and the public.
+
+## Install
+
+This example should Using lazy you can use this:
```

Comment:
non-blocking

I think there's a typo here. Did you mean something like `"Here is an example using lazy dependencies:"`?

Reply:
Yes, you are right, thanks for reviewing in detail!